x-pack/plugins/lens/public/datatable_visualization/components/cell_value.tsx

```diff
@@ -53,18 +53,6 @@ export const createGridCell = (
           }
         }
       }
-      // make sure to clean it up when something change
-      // this avoids cell's styling to stick forever
-      return () => {
-        if (minMaxByColumnId?.[originalId]) {
-          setCellProps({
-            style: {
-              backgroundColor: undefined,
-              color: undefined,
-            },
-          });
-        }
-      };
```

Comment:
cc @dej611 on this change: https://github.com/elastic/eui/issues/4665#issuecomment-904109739

I tested locally and I'm fairly sure this is still working with the hook cleanup gone:

![lens](https://user-images.githubusercontent.com/549407/130708544-d2b23b3b-ccfa-42a0-b0ad-c20b5d86c357.gif)

Feel free to double check/confirm yourself of course! 🙏

Reply:
The fix does work ok in that specific case, but it broke the table when removing the conditional coloring:

![not_cleaning_cells](https://user-images.githubusercontent.com/924948/131466997-eb1e7ccd-2a6c-48e1-b882-e05eab9f2ae1.gif)

Reply:
Keeping this code here the behaviour works as expected.

Reply:
Super strange! Will definitely revert this until we get it figured out. Thank you for QAing @dej611!